src/causalprog/graph/node.py

```diff
@@ -143,7 +141,7 @@ class DistributionNode(Node):
 
     def __init__(
         self,
-        distribution: DistributionFamily,
+        distribution: type,
```

Comment:
There's probably a more helpful typehint we can put here (with enough effort), but that can be a bonus job for https://github.com/UCL/causalprog/issues/27